microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc

```diff
@@ -8,6 +8,13 @@ toc::[]
 
 Embedding {microshift-short} containers in an `rpm-ostree` commit means that you can run a cluster in air-gapped, disconnected, or offline environments. You can embed {product-title} containers in a {op-system-ostree-first} image so that container engines do not need to pull images over a network from a container registry. Workloads can start up immediately without network connectivity.
 
+Plan to use a supported version of {op-system} as listed in the following compatibility table of the "About {product-title} updates section" link.
+
+//additional resources for system requirements/compatibility table
+[role="_additional-resources"]
+.Additional resources
+* xref:../microshift_updating/microshift-about-updates.adoc#microshift-about-updates[About {product-title} updates]
```

Comment:
Readers usually resent being sent to another doc set. How about we just plop the table right in here? (Bear in mind, this version of the table is not updated yet--it will update with each branch-specific PR.)